default.nix

```diff
@@ -0,0 +1,12 @@
+let pkgs = import <nixpkgs> {};
+in { stdenv ? pkgs.stdenv, xz ? pkgs.xz }:
+stdenv.mkDerivation {
+  name = "slippc";
+  src = ./.;
+  buildDepends = [ xz ];
```

Comment:
This is `liblzma`

https://github.com/NixOS/nixpkgs/blob/2b7c73e4a6ad20fac26defc8a657ab2072387fd7/pkgs/tools/compression/xz/default.nix#L88